example/spec/parallel/ingest/ingestFromPdrSpec.js

```diff
@@ -256,9 +256,15 @@ describe('Ingesting from PDR', () => {
           queueGranulesOutput.payload.running
             .map((arn) => waitForCompletedExecution(arn))
         );
+        await Promise.all(parseLambdaOutput.payload.granules.map(
+          (granule) => granulesApiTestUtils.deleteGranule({
+            prefix: config.stackName,
+            granuleId: granule.granuleId,
+          })
+        ));
         await granulesApiTestUtils.deleteGranule({
           prefix: config.stackName,
-          granuleId: parseLambdaOutput.payload.granules[0].granuleId,
+          granuleId: testDataGranuleId,
```

Comment:
Shouldn't this be covered by the statement on L259?

Reply:
Yeah, thanks for finding that!

Reply:
[`815243d` (#2254)](https://github.com/nasa/cumulus/pull/2254/commits/815243d4a145724b531e1380b1140e63f738bb96)